_shows/16_17/crystal_quest.md

```diff
@@ -39,6 +39,17 @@ prod_shots: bjxVx7
 assets:
   - type: poster
     image: r3JPbCh
+    
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2017/06/crystal-quest-5d-adventure-nottingham-new-theatre/
+    snapshot: E7Lcm
+    publisher: Impact Nottingham
+    author: Emma Pallen
+    title: "Crystal Quest: A 5D Adventure @ Nottingham New Theatre"
+    date: 2017-06-15
+    rating: 10/10
+    quote: “Crystal Quest: A 5D Adventure is a one-of-a-kind production that can be watched again and again”
```

Comment:
Actual problem: The quote marks you've used are incorrect. Unsure what editor you're using but stylised quote marks have snuck in: `“Crystal ... again”` should be `"Crystal ... again"`.

(Techincal background about quotemarks here https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html, the table at the top shows the various possible symbols)

Reply:
Ah, when I said I lifted the pull-quote from the Impact site, it came with the quote marks. Good to know for the future.

Reply:
YAML issues tend to be a bit of a PITA. 

For reference: I shoved the frontmatter block (everything between the `---` marks, inclusive) into http://www.yamllint.com/ which points out where the issue is.

![image](https://user-images.githubusercontent.com/1690934/31856432-e41dac30-b6b8-11e7-9d28-b6f6b0b5f401.png)